objgraph.py

```diff
@@ -424,6 +510,30 @@ def at(addr):
     return None
 
 
+def at_addrs(address_set):
+    """Return objects for a given set of memory addresses.
+
+    The reverse of id(obj):
+
+        >>> at(id(obj)) is obj
+        True
+
+    Note that this function does not work on objects that are not tracked by
+    the GC (e.g. ints or strings). A good use of this function is to call:
+
+        >>> [old, current, new_ids] = get_ids()
+        new_lists = at_addrs(new_ids['list'])
+        for new_list in new_lists:
+            call show_chain on each new_list object
```

Comment:
Doctests need a `>>>` in front of every statement, and a `...` in front of every continuation line of a compound statement such as a `for` loop.

Doctests are executed by the test runner, and so cannot contain pseudocode.